source/features/convert-pr-to-draft-improvements.tsx

```diff
@@ -14,6 +14,7 @@ function closeModal({delegateTarget: button}: DelegateEvent<MouseEvent, HTMLButt
 }
 
 function addConvertToDraftButton(alternativeActions: Element): void {
+	alternativeActions.classList.add('d-inline-flex', 'gap-1');
```

Comment:
Same as https://github.com/refined-github/refined-github/pull/8159#issuecomment-2550558616

In this case it seems you only need to call `convertToDraft.classList.replace('d-inline', 'd-inline-block')`

Reply:
OOOOOOOOOO